admin/package-files/systemd/flocker-agent.service

```diff
@@ -1,11 +1,11 @@
 [Unit]
-Description=Flocker ZFS Agent
+Description=Flocker Dataset Agent
 AssertFileNotEmpty=/etc/flocker/agent.yml
 
 [Service]
-ExecStart=/usr/sbin/flocker-zfs-agent
+ExecStart=/usr/sbin/flocker-dataset-agent
 
-# We can't have a private mount namespace, since we need to see ZFS created mounts.
+# We can't have a private mount namespace, since we need to create mounts visible to Docker.
```

Comment:
This comment seems a bit...random...but I think it means that we can't add an extra systemd configuration directive to limit the mount points visible to the process that we're launching. Has @tomprince got a link to the issue that he fixed a few months ago where the Docker service file *was* limiting dockerd to a private namespace?